### PR TITLE
new post pathway created to add new comment

### PR DESCRIPTION
**Pull request by DJHymers**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -217,6 +217,46 @@ describe("nc_games_test", () => {
...
+    it("400: failing schema validation", () => {
+      return request(app)
+        .post("/api/reviews/3/comments")
+        .send({ username: 123, body: 654 })
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Invalid input");
+        });
+    });
+  });
```
Comment (robert112233) [inline __tests__/app.test.js:259]: It's still worth testing the usual errors here, 400 invalid review ID,  404 review ID not found etc. 

You should also include a test for 404 invalid username {username: "aUserThat IsntInTheDatabase", body : "validComment"}

**Review by robert112233 — CHANGES_REQUESTED** [review]
Write the missing tests then submit another PR for review, nice work 👍

File: __tests__/app.test.js [hunk 2]
```diff
@@ -217,6 +217,82 @@ describe("nc_games_test", () => {
...
+        .send({
+          username: "mallionaire",
+          body: "I love hosting",
+        })
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("404 path not found");
+        });
+    });
+    it("400: invalid username provided, should provide bad path message", () => {
```
Comment (RichardNeat) [inline __tests__/app.test.js:283]: a 404 would be more appropriate here as the request isn't 'bad' necessarily it's just that it doesn't exist.

**Review by RichardNeat — APPROVED** [review]
Nice one David, just one minor change I've mentioned below then feel free to merge :)

**Pull request merged**